### PR TITLE
[json-schema] Added schema for performance monitoring notifications

### PR DESCRIPTION
**Pull request by mpastorek**
### Code of Conduct
- [ ] Title of the PR starts with chart name (e.g. `[inventory]`)
- [ ] Update package version in principles of semantic versioning
- [ ] Check RELEASE.md 
- [ ] Update CHANGELOG.md

### Type of Change
- [ ] **Feature**: A new addition to the project that requires a MINOR version bump.
- [ ] **Fix**: A bug fix that requires a PATCH version bump.
- [ ] **Breaking Change**: Any change that is NOT backward-compatible, requiring a MAJOR version bump.

### Versioning Strategy
Explain how the version number was determined based on semantic versioning principles.

### References
- [Semantic Versioning 2.0.0](https://semver.org/)

### CODE REVIEW

**Comment by Jozefiel** [comment]
please bump the package version to 0.1.0 because new feature was added

File: utils/json-schemas/performance_notification.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      "type": "string"
+    },
+    "event_time": {
+      "description": "The timestamp when notification arrived.",
+      "type": "string"
+    },
+    "body": {
+      "description": "The data.",
+      "type": "string"
+    }
```
Comment (jaro0149) [inline utils/json-schemas/performance_notification.json:23]: please, specify which fields are mandatory

File: utils/json-schemas/performance_notification.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      "description": "The name of the node.",
+      "type": "string"
+    },
+    "identifier": {
+      "description": "Identifier/path to data.",
+      "type": "string"
+    },
+    "event_time": {
+      "description": "The timestamp when notification arrived.",
+      "type": "string"
```
Comment (jaro0149) [inline utils/json-schemas/performance_notification.json:18]: there are already some built-in timestamp formats: https://json-schema.org/understanding-json-schema/reference/string#dates-and-times

does it fit one of them?
if not, you can still define pattern-based constraint

File: utils/json-schemas/performance_notification.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    },
+    "body": {
+      "description": "The data.",
+      "type": "string"
```
Comment (jaro0149) [inline utils/json-schemas/performance_notification.json:22]: are these events sent as escaped jsons? if not, then 'object' type is better:
https://json-schema.org/understanding-json-schema/reference/object

**Review by jaro0149 — CHANGES_REQUESTED** [review]

File: utils/json-schemas/performance_notification.json (hunk 2 above)
Comment (mpastorek) [reply utils/json-schemas/performance_notification.json]: this is the timestamp from notification:

File: utils/json-schemas/performance_notification.json (hunk 3 above)
Comment (mpastorek) [reply utils/json-schemas/performance_notification.json]: Probably unescaped. But now I have only notifications with numbers like this:
    "body": {
        "nokia-state:summary": {
            "available-memory": "2297430016",
            "current-total-size": "1862270976",
            "total-in-use": "1679717632"
        }
    }
I changed field to object

**Review by jaro0149 — APPROVED** [review]

**Pull request merged**